eo-maven-plugin/src/test/java/org/eolang/maven/BinarizeMojoTest.java

```diff
@@ -72,4 +75,25 @@ void failsWithIncorrectInsert(@TempDir final Path temp) throws IOException {
             () -> maven.execute(new FakeMaven.Binarize())
         );
     }
+
+    @Test
+    @Tag("slow")
```

Comment:
@levBagryansky Could you please describe what is the purpose of this annotation ? E.g. without `@Suite` and `@IncludeTags` annotations?

Reply:
@Graur this is from #2091. This test is executed in `Pqulice` only.

Reply:
@levBagryansky I see. Thanks!